kahuna/public/js/components/gu-lazy-gallery-shortcuts/gu-lazy-gallery-shortcuts.js

```diff
@@ -0,0 +1,40 @@
+import angular from 'angular';
+
+import '../gr-keyboard-shortcut/gr-keyboard-shortcut';
+
+export var lazyGalleryShortcuts = angular.module('gu.lazyGalleryShortcuts', [
+    'gr.keyboardShortcut'
+]);
+
+lazyGalleryShortcuts.directive('guLazyGalleryShortcuts',
+                             ['keyboardShortcut',
+                              function(keyboardShortcut) {
+    return {
+        restrict: 'EA',
+        require: '^guLazyGallery',
+        link: function (scope, element, attrs, lazyGalleryCtrl) {
+            console.log(lazyGalleryCtrl);
+            function invoke(fnName) {
+                return (event) => {
+                    // Must cancel any scrolling caused by the key
+                    event.preventDefault();
+                    lazyGalleryCtrl[fnName]();
```

Comment:
I meant that it didn't call the function. I've since worked out it's because the names of the functions were wrong and Im a moron.